tests/e2e-evm/test/helpers/abi.ts

```diff
@@ -0,0 +1,21 @@
+import { Abi, AbiFallback, AbiReceive } from "abitype";
+
+export function getAbiFallbackFunction(abi: Abi): AbiFallback | undefined {
+  for (const f of abi) {
```

Comment:
This is the golang dev in me -- I probably should just use Array#find or similar, tho maybe that's more difficult for typescript to infer the type 🤔